pkg/trace/api/profiles.go

```diff
@@ -76,6 +109,48 @@ func newProfileProxy(transport http.RoundTripper, target *url.URL, apiKey, tags
 	return &httputil.ReverseProxy{
 		Director:  director,
 		ErrorLog:  stdlog.New(logger, "profiling.Proxy: ", 0),
-		Transport: transport,
+		Transport: &multiTransport{transport, targets, keys},
+	}
+}
+
+type multiTransport struct {
```

Comment:
Can we document what this does?
```suggestion
// multiTransport sends all requests to multiple targets and API keys using an
// underlying http.RounTripper.
type multiTransport struct {
```